move-mutator/doc/design.md

```diff
@@ -201,7 +201,6 @@ operators = ["operator1", "operator2"]
 file = "/path/to/file"
 verify_mutants = true
 include_functions = ["function1", "function2"]
-exclude_functions = ["function3", "function4"]
```

Comment:
This is a nice feature, but it's not implemented in the code.

This doc might get a few more updates in the future